src/ints/mouse.cpp

```diff
@@ -453,14 +453,14 @@ void MouseQueue::StartTimerIfNeeded()
     bool timer_needed = false;
     uint8_t delay_ms  = UINT8_MAX;
 
-    if (delay.ps2_ms) {
+    if (HasEventPS2()) {
         timer_needed = true;
         delay_ms     = std::min(delay_ms, delay.ps2_ms);
     }
-    if (delay.dos_moved_wheel_ms) {
+    if (HasEventDosMoved()) {
         timer_needed = true;
         delay_ms     = std::min(delay_ms, delay.dos_moved_wheel_ms);
-    } else if (delay.dos_button_ms) {
+    } else if (HasEventDosButton()) {
```

Comment:
If the user moves and clicks at the same time, I guess the `dos_moved_wheel_ms` will always be used as the delay - but both the move and click will still be registered?

Reply:
Sorry, I see this is part of the queue processing, so after digesting the move event the button will come next.